### PR TITLE
feat(Context-Based Restrictions): add service group support to Operations

### PR DESCRIPTION
**Pull request by JonahFarc**
## PR summary
Adds service group support to operations
Should replace https://github.com/IBM/platform-services-node-sdk/pull/223

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [X] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  
service_name parameter can now be replaced with the service group parameter.

## Does this PR introduce a breaking change?    
- [ ] Yes
- [X] No

### CODE REVIEW

**Review by soohoowoohoo — APPROVED** [review]
LGTM

**Comment by JonahFarc** [comment]
test screenshot:
image

Examples Screenshot:
image

File: context-based-restrictions/v1.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ import { getSdkHeaders } from '../lib/common';
  */
 
 class ContextBasedRestrictionsV1 extends BaseService {
-  static DEFAULT_SERVICE_URL: string = 'https://cbr.cloud.ibm.com';
+  static DEFAULT_SERVICE_URL: string = 'https://cbr.test.cloud.ibm.com';
```
Comment (padamstx) [inline context-based-restrictions/v1.ts:42]: You must have used the staging version of the API definition when you generated this code.  I didn't see this change in the other PRs.   Please re-gen with the same version of the API definition that you used for the other languages.

**Review by padamstx — CHANGES_REQUESTED** [review]
Need to re-gen with the same API definition file as the other languages.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**